docs/data-import-and-processing.adoc

```diff
@@ -62,31 +62,30 @@ This means that you can have `update_other` turned on and `update_parallels` dis
 
 == Synchronization
 
-The external data can be changed or deleted at any time, so the synchronization process has to account for that.
-KOS(api) does not provide any diffs or changesets, so full synchronization has to be performed every time an update is required.
-Synchronization is usually scheduled multiple times a day, so that the most recent data are provided.
+The external data can be changed or deleted any time, so the synchronization process has to account for that.
+KOS(api) does not provide any diffs or changesets, so we need to performa a full synchronization every time an update is required.
+Synchronization happens multiple times a day to provide up-to-date data.
 
-When synchronizing with existing database data, events are generated as usual and when saving them, an attempt is made to match them with the existing events in the database.
+We synchronize existing database data as follows: events are generated as usual. When the generated events are saved to dabase, an attempt is made to match them with the existing events in the database.
```

Comment:
This sentence is kinda weird: “We synchronize existing database data **as follows: events are generated as usual.**”

Typo: …are saved to **dabase**…